.dockerignore

```diff
@@ -0,0 +1 @@
+frontend
```

Comment:
This prevents the backend container Dockerfile (which this PR moves to the top level directory) from having to pull in the frontend files during build.